R/functions.R

```diff
@@ -54,7 +54,23 @@ get_planning_raw <- function(ss = gs_id()) {
   read_sheet(
     ss,
     sheet = "Planning_v2",
-    col_types = "ccccclllilccccccdddddddddddccdddddddddddcc",
+    range = cell_cols("A:AP"),
+    col_types =
+      c(
+        rep("c", 5),   # task definition
+        rep("l", 3),   # task properties
+        "i",           # priority
+        "l",           # selected for execution
+        rep("c", 5),   # management stuff
+        "c",           # who executes
+        rep("d", 11),  # number of days per team member
+        "c",           # who else executes
+        "c",           # who reviews
+        rep("d", 11),  # number of days per team member
+        "c",           # who else reviews
+        "c"            # total number of teamdays
+      ) |>
+      str_flatten(),
```

Comment:
This function is from `stringr`, I think? 
The library is not loaded per default, which causes a "not found" error. I suggest adjusting to `stringr::str_flatten()`.

Reply:
Thanks for reviewing! (I didn't want to bother you with this)

Well, {stringr} is loaded by the `R/planning_helper.R` script in line 15, which is the script that calls the functions. So from the viewpoint from the script, there's no problem. But if we were to make it a true standalone function, then you're absolutely right (and more similar changes would be needed). It was not my initial purpose to harden it this much.

Reply:
(This occurred to me in a different workflow: the "planning backup". Simple to resolve.)